add-paths nil pointer dereference

Hi, when I try adding a new path, I get the following error:

```
$ rly config add-paths nchainz/config/paths/
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x4edf3bf]

goroutine 1 [running]:
github.com/cosmos/relayer/relayer.(*Chain).QueryClientState(0xc000f5c240, 0x702, 0x9, 0x40, 0x0)
        /Users/michael/agoric/forks/relayer/relayer/query.go:130 +0x7f
github.com/cosmos/relayer/cmd.(*Config).ValidateClient(0xc000558780, 0xc000f5c240, 0x702, 0xc000f815e0, 0x4, 0xc000f5c240)
        /Users/michael/agoric/forks/relayer/cmd/config.go:551 +0x75
github.com/cosmos/relayer/cmd.(*Config).ValidatePathEnd(0xc000558780, 0xc000f815e0, 0x20, 0x4fe0ae0)
        /Users/michael/agoric/forks/relayer/cmd/config.go:517 +0x1c5
github.com/cosmos/relayer/cmd.(*Config).ValidatePath(0xc000558780, 0xc000fbecc0, 0x51deeb5, 0x1)
        /Users/michael/agoric/forks/relayer/cmd/config.go:479 +0x5c
github.com/cosmos/relayer/cmd.cfgFilesAddPaths(0x7ffeefbff42d, 0x14, 0xc000e5fd50, 0x0, 0x0)
        /Users/michael/agoric/forks/relayer/cmd/config.go:287 +0x4e5
github.com/cosmos/relayer/cmd.configAddPathsCmd.func1(0xc000f1d080, 0xc000f09e30, 0x1, 0x1, 0x0, 0x0)
        /Users/michael/agoric/forks/relayer/cmd/config.go:206 +0x45
github.com/spf13/cobra.(*Command).execute(0xc000f1d080, 0xc000f09e10, 0x1, 0x1, 0xc000f1d080, 0xc000f09e10)
        /Users/michael/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:850 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc000f1c2c0, 0x0, 0x0, 0x0)
        /Users/michael/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/michael/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
github.com/cosmos/relayer/cmd.Execute()
        /Users/michael/agoric/forks/relayer/cmd/root.go:116 +0x3d
main.main()
        /Users/michael/agoric/forks/relayer/main.go:21 +0x25
```

The `nchainz/config/paths` directory contains only a single file, `path-1.json`, whose contents are:

```sh
{
  "src": {
    "chain-id": "ibc0",
    "client-id": "07-tendermint-0",
    "connection-id": "connection-0",
    "port-id": "port-1",
    "order": "ordered",
    "version": "echovsn"
  },
  "dst": {
    "chain-id": "ibc1",
    "client-id": "07-tendermint-0",
    "connection-id": "connection-0",
    "port-id": "echo",
    "order": "ordered",
    "version": "ics20-1"
  },
  "strategy": {
    "type": "naive"
  }
}
```

Output from `rly config list` (note that `path-0` is currently being streaming relayed, but I've reproduced the failure even when the relayer is stopped):

```yaml
global:
  api-listen-addr: :5183
  timeout: 10s
  light-cache-size: 20
chains:
- key: testkey
  chain-id: ibc0
  rpc-addr: http://localhost:26657
  account-prefix: agoric
  gas-adjustment: 1
  gas-prices: ""
  trusting-period: 336h
- key: testkey
  chain-id: ibc1
  rpc-addr: http://localhost:26557
  account-prefix: agoric
  gas-adjustment: 1
  gas-prices: ""
  trusting-period: 336h
paths:
  path-0:
    src:
      chain-id: ibc0
      client-id: 07-tendermint-0
      connection-id: connection-0
      channel-id: channel-0
      port-id: transfer
      order: unordered
      version: ics20-1
    dst:
      chain-id: ibc1
      client-id: 07-tendermint-0
      connection-id: connection-0
      channel-id: channel-0
      port-id: transfer
      order: unordered
      version: ics20-1
    strategy:
      type: naive
```

--- a/cmd/config.go
+++ b/cmd/config.go
@@ -507,6 +507,11 @@ func (c *Config) ValidatePathEnd(pe *relayer.PathEnd) error {
 	if err != nil {
 		return err
 	}
+	// NOTE: this is just to do validation, the path
+	// is not written to the config file
+	if err = chain.SetPath(pe); err != nil {
+		return err
+	}
 
 	height, err := chain.QueryLatestHeight()
 	if err != nil {

